### ISSUE TITLE
Error when passing a string to OmegaNum that exceeds 1000 arrows (OmegaNum.maxArrow default)

### ISSUE THREAD

**Issue by sonic3XE** [Naruyoko/OmegaNum.js#39]
This evaluation doesn't warn and/or give Infinity because of the error occurred before the evaluation is complete.
```
OmegaNum("10{1000}10")
OmegaNum.js:1256 Uncaught TypeError: arrows.add is not a function
    at Q.fromString (OmegaNum.js:1256:87)
    at new OmegaNum (OmegaNum.js:1465:23)
    at OmegaNum (OmegaNum.js:1445:44)
    at <anonymous>:1:1
```
Other methods work fine as since
```
OmegaNum.arrow(10,1000,10).eq(Infinity)
OmegaNum.js:877 Number too large to reasonably handle it: tried to 1002-ate.
(anonymous) @ OmegaNum.js:877
Q.arrow @ OmegaNum.js:919
(anonymous) @ VM323:1
true
```

**Comment by Naruyoko** [comment]
Fix released in [0.7.0](https://github.com/Naruyoko/OmegaNum.js/releases/tag/v0.7.0)

**Comment by sonic3XE** [comment]
This is confirmed that v0.7.0 fixed this issue.
![image](https://github.com/Naruyoko/OmegaNum.js/assets/137403814/8d1ed807-14dd-4d70-953c-33ac633c8fdb)
